contracts/examples/adder/src/example_interactor_gen.rs

```diff
@@ -0,0 +1,127 @@
+use adder::ProxyTrait as _;
```

Comment:
This file is not generated by the generator, I'd say remove.

Better to have a test that compares output with a committed source.